ui/components/app/app-components.scss

```diff
@@ -51,7 +51,7 @@
 @import 'recovery-phrase-reminder/index';
 @import 'step-progress-bar/index.scss';
 @import 'selected-account/index';
-@import 'smart-transactions/index';
+@import '../../pages/smart-transactions/components/index.scss';
```

Comment:
I don't see any other import from `../../pages` in this component. Shouldn't we remove it from here and import it somewhere else?

Reply:
Good call!